File: beakjoon/C++/2309.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main() {
	vector<int> arr;
	int sum = 0;
	for (int i = 0; i < 9; i++) {
		int a;
		cin >> a;
		sum += a;
		arr.push_back(a);
	}

	int a = 0;
	int b = 0;

	for (int i = 1; i < 9; i++) {
		for (int j = 0; j < 8; j++) {
			if (sum - arr[i] - arr[j] == 100) {
				a = i;
				b = j;
			}
		}
	}

	arr.erase(arr.begin() + a);
	arr.erase(arr.begin() + b);
	sort(arr.begin(), arr.end());

	for (int i = 0; i < arr.size(); i++) {
		cout << arr[i] << endl;
	}
	return 0;
}

File: beakjoon/python/2309.py
lst = []

for i in range(9):
    lst.append(int(input()))


for i in range(1,9):
        for j in range(0,8):
            if (sum(lst) - lst[i] - lst[j]) == 100:
                a = i
                b = j

del lst[a]
del lst[b]

lst.sort()

for i in lst:
    print(i)